packages/react/src/utils/ipfs.ts

```diff
@@ -15,18 +16,93 @@ export const checkIpfsAvailability = async (url: string): Promise<string> => {
 
   const cid = url.replace('ipfs://', '')
   const nfdUrl = `https://images.nf.domains/ipfs/${cid}`
```

Comment:
we'll need to point to something else in the future but definitely fine for now.